sdk/storage/src/datalake/datalake_utilities.cpp

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#include "datalake/datalake_utilities.hpp"
+
+#include "common/crypt.hpp"
+#include "datalake/protocol/datalake_rest_client.hpp"
+
+#include <regex>
```

Comment:
CANNOT use regex, GCC 4.8.5 is in our platform matrix. std::regex isn't supported until GCC 5

Reply:
A bigger problem is that the CI hasn't caught this, I do vaguely remember you talked about this. Sorry about that.

Reply:
Plus I do think it's overkill to do such a easy job with regex.